package.json

```
@@ -180,7 +180,7 @@
     "outputName": "junit.xml"
   },
   "dependencies": {
-    "@atala/apollo": "^1.2.16",
+    "@atala/apollo": "^1.3.4",
```

I think maybe the actual change for this PR should have been the below, then we wouldn't have to change any other files
```suggestion
    "@atala/apollo": "1.2.16",
```